code/API_definitions/device-data-volume-subscriptions.yaml

```diff
@@ -0,0 +1,1244 @@
+            subscriptionIdRequired:
+              summary: subscription id is required
+              value:
+                status: 400
+                code: INVALID_ARGUMENT
+                message: "Expected property is missing: subscriptionId"
+
+  examples:
+    DATA_50_PERCENT:
+      value:
+        id: "124007"
+        source: https://dataUsageServer.supertelco.com
+        type: org.camaraproject.device-data-volume-subscriptions.v0.data-50-percent
+        specversion: "1.0"
+        datacontenttype: application/json
+        data:
+          device:
+            phoneNumber: "+123456789"
+          dataUsageThreshold: "50%"
+          subscriptionId: "dv10-h556-rt89-1406"
+        time: "2024-05-12T12:45:23.682Z"
+
+    DATA_75_PERCENT:
+      value:
+        id: "124008"
+        source: https://dataUsageServer.supertelco.com
+        type: org.camaraproject.device-data-volume-subscriptions.v0.data-75-percent
+        specversion: "1.0"
+        datacontenttype: application/json
+        data:
+          device:
+            phoneNumber: "+123456789"
+          dataUsageThreshold: "75%"
+          subscriptionId: "dv10-h556-rt89-1407"
+        time: "2024-05-12T14:30:23.682Z"
+
+    DATA_90_PERCENT:
+      value:
+        id: "124009"
+        source: https://dataUsageServer.supertelco.com
+        type: org.camaraproject.device-data-volume-subscriptions.v0.data-90-percent
+        specversion: "1.0"
+        datacontenttype: application/json
+        data:
+          device:
+            phoneNumber: "+123456789"
+          dataUsageThreshold: "90%"
+          subscriptionId: "dv10-h556-rt89-1408"
+        time: "2024-05-12T16:15:23.682Z"
+
+    DATA_EXCEEDED:
+      value:
+        id: "124010"
+        source: https://dataUsageServer.supertelco.com
+        type: org.camaraproject.device-data-volume-subscriptions.v0.data-exceeded
+        specversion: "1.0"
+        datacontenttype: application/json
+        data:
+          device:
+            phoneNumber: "+123456789"
+          dataUsageThreshold: "100%"
+          subscriptionId: "dv10-h556-rt89-1409"
+        time: "2024-05-12T18:00:23.682Z"
+
+    SUBSCRIPTION_ENDS:
+      value:
+        id: "124003"
+        source: https://dataUsageServer.supertelco.com
+        type: org.camaraproject.device-data-volume-subscriptions.v0.subscription-ends
+        specversion: "1.0"
+        datacontenttype: application/json
+        data:
+          device:
+            phoneNumber: +123456789
+          terminationReason: SUBSCRIPTION_EXPIRED
+          subscriptionId: dv10-h556-rt89-1403
+        time: 2024-03-05T15:00:23.682Z
+
+    SUBSCRIPTION_ACTIVE:
+      value:
+        id: 550e8400-e29b-41d4-a716-446655440001
+        sink: https://endpoint.example.com/sink
+        protocol: HTTP
+        types:
+          - "org.camaraproject.device-data-volume-subscriptions.v0.data-usage-exceeded"
+        config:
```

Comment:
It should be 'data-exceeded' only. There is no reference of 'data-usage-exceeded'